pkgs/tools/security/pass/extensions/tail.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "pass-tail";
+  version = "1.2.0";
+
+  src = fetchFromGitHub {
+    owner = "palortoff";
+    repo = "pass-extension-tail";
+    rev = "v${version}";
+    sha256 = "RyPXW4Lgx6GdF9j+zEJqKRGxinDwPCmF9WXJ5yW4Pcc=";
+  };
+
+  buildInputs = [ ];
+
+  dontBuild = true;
+
+  installFlags = [
+    "PREFIX=$(out)"
+    "BASHCOMPDIR=$(out)/share/bash-completion/completions"
+  ];
+
+  meta = with lib; {
```

Comment:
You can easily remove `with lib;` here.

Reply:
I added it because if not, I would have to repeat it on each attribute:

```nix
  meta = {
    description = "A pass extension to avoid printing the password to the console";
    homepage = "https://github.com/palortoff/pass-extension-tail";
    license = lib.licenses.gpl3;
    maintainers = with lib.maintainers; [ pablito2020 ];
    platforms = lib.platforms.unix;
  };
```

So I thought that adding it made the code "cleaner":

```nix 
meta = with lib; {
    description = "A pass extension to avoid printing the password to the console";
    homepage = "https://github.com/palortoff/pass-extension-tail";
    license = licenses.gpl3;
    maintainers = with maintainers; [ pablito2020 ];
    platforms = platforms.unix;
};

```

I can agree with the "with maintainers" one.

Reply:
Yes, the idea is to completely remove it when it is in front of a set, just like it's done on Nixos/modules since a couple of months now.

Reply:
Okey, may I ask why? I mean in my eyes adding "with lib;" makes the code easier to read, but I can see that when you have a lot of attributes you don't know which ones are comming from lib and which ones are not.